### PR TITLE
Music button

### PR DESCRIPTION
**Pull request by habraham2023**
Resolves: #11

- Music is now played when clicking on volume icon.
- Music will pause when clicked again.
- Music state is a single instance so when switching between pages music isn't interrupted.

### CODE REVIEW

**Review by aileenpongnon — CHANGES_REQUESTED** [review]
amazing work @habraham2023 @leahhowells! it's so nice.

songs to be used for hackathon are located in the Design notion page under "Music (Zen Garden Website)
https://www.notion.so/Music-Zen-Garden-Website-2021-767f396d37c246a8825176c2ff8a061e

next step should be implementing these. one more thing I'm thinking of is having a "Playing... [name of song]" somewhere on the page but that should be saved for a post-launch enhancement.

File: src/components/VolumeButton.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useContext } from "react";
+import { IconContext } from "react-icons";
+import { BsFillVolumeUpFill, BsFillVolumeMuteFill } from "react-icons/bs";
+import { VolumeContext } from "../context/VolumeContext";
+
+const VolumeButton = () => {
+  const volume = useContext(VolumeContext);
+  console.log(volume);
```
Comment (rob-3) [inline src/components/VolumeButton.js:8]: This should be removed

**Review by rob-3 — CHANGES_REQUESTED** [review]
Looks mostly good. Something I noticed in the `frontend-dev` branch and here is a lot of duplicated use of `<Helmet />`. Any reason we can refactor react-helmet into `<Page />` and just pass `title` as a prop?

**Review by Ro-Hanna — COMMENTED** [review]
Looks great, the transition in both audio and visual when pressing for a song to stop/play is smooth. I would also suggest incorporating text that shows what song is playing post-deployment.

**Pull request merged**